retail/retail-java-applications/terraform/test/pubsub_test.go

```diff
@@ -0,0 +1,90 @@
+/**
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package test
+
+import (
+	"cloud.google.com/go/pubsub"
+	"context"
+	"fmt"
+	"github.com/gruntwork-io/terratest/modules/gcp"
+	"github.com/gruntwork-io/terratest/modules/random"
+	"github.com/gruntwork-io/terratest/modules/terraform"
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func TestPubsub(t *testing.T) {
+	projectId := gcp.GetGoogleProjectIDFromEnvVar(t)
+	opts := &terraform.Options{
+		TerraformDir: "../examples/pubsub",
+		Vars:         map[string]interface{}{"project_id": projectId},
+	}
+	defer terraform.Destroy(t, opts)
+	terraform.Init(t, opts)
+	terraform.Apply(t, opts)
+	clickstreamInboundTopicName := "Clickstream-inbound"
+	inventoryInboundTopicName := "Inventory-inbound"
+	transactionsInboundTopicName := "Transactions-inbound"
+	clickstreamInboundSubName := "Clickstream-inbound-sub"
+	inventoryInboundSubName := "Inventory-inbound-sub"
+	transactionsInboundSubName := "Transactions-inbound-sub"
+	ctx := context.Background()
+	client, err := pubsub.NewClient(ctx, projectId)
+	if err != nil {
+		fmt.Println(err)
+	}
+	// clickstream inbound pubsub
+	expectedCI := fmt.Sprintf("clickstream-inbound-test-message-%s", random.UniqueId())
```

Comment:
expectedCI is not an intuitive name. Same for expectedII and expectedTI. Use expected or something else clearer